src/main/java/org/spongepowered/api/entity/Entity.java

```diff
@@ -531,4 +531,13 @@ default boolean canSee(Entity entity) {
         Optional<Boolean> optional = entity.get(Keys.VANISH);
         return !optional.isPresent() || !optional.get();
     }
+
+    /**
+     * Returns whether this entity has gravity.
+     *
+     * @return True if this entity has gravity
+     */
+    default boolean hasGravity() {
```

Comment:
I'd keep it as `gravity()`. Refer to other Entity based interfaces where the default methods are included and such.